nav/backplanes/merge.py

```
@@ -0,0 +1,133 @@
+
+    # Bodies: scalar distances per body, broadcast within mask; inf elsewhere
+    for body_name, entry in bodies_result['per_body'].items():
+        distance_scalar = float(entry['distance'])
+        # Create per-pixel distance with inf default and scalar where within any mask
+        any_mask = None
+        for m in entry['masks'].values():
+            if any_mask is None:
+                any_mask = m
+            else:
+                if not np.all(any_mask == m):
+                    raise ValueError(f'Masks for body {body_name} are not all the same')
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)